articles/function-declarations-and-calls.html

```diff
@@ -132,7 +132,7 @@ <h3>Function Declarations</h3>
 
 <p>
 If the result declaration list in a function declaration only contains
-one anonymous result declaration, then the result declaration list doesn't
+one anonymous result declaration, then the result declaration list doesn't.

```

Comment:
It was my fault that I mis-deleted a line in the last commit.
Here it should be 
```
one anonymous result declaration, then the result declaration list doesn't
need to be enclosed in a <code>()</code>.
```

Reply:
Got it